browserexport/browsers/chrome.py

```diff
@@ -50,7 +51,7 @@ def extract_visits(cls, path: PathIshOrConn) -> Iterator[Visit]:
             )
 
     @classmethod
-    def data_directory(cls) -> Path:
+    def data_directory(cls) -> Paths:
```

Comment:
the name is now somewhat misleading, let me know if you want to switch to plural

Reply:
Hmm - yeah, can rename it. Can add a @classmethod in the base `Browser` implementation in `common.py` like:

```
@classmethod
def data_directory(cls) -> Paths:
    # send deprecation warning
    return cls.data_directories() 
```
-- will remove it in a few months then

Reply:
Ah, would probably still break something since people are expecting the single path and not a sequence. Could return the first path from `cls.data_directories()` in that

Reply:
ah yeah, I was thinking of asserting if it's not a single element list? or you'd rather return first path so it doesn't crash?

Reply:
In the deprecated funciton, would rather just return the first path/send a warning if there are multiple, just so it doesn't crash